valohai/inputs.py

```diff
@@ -1,14 +1,15 @@
-from typing import IO, Iterable, Iterator, Optional
+from typing import IO, Iterable, Iterator, List, Optional, Union
 
 from valohai.internals import vfs
 from valohai.internals.download_type import DownloadType
-from valohai.internals.input_info import load_input_info
+from valohai.internals.inputs import get_input_vfs
 from valohai.paths import get_inputs_path
 
 
 class Input:
-    def __init__(self, name: str) -> None:
+    def __init__(self, name: str, default: Union[str, List[str]] = None) -> None:
```

Comment:
If the default is None, this should probably be `Optional`...
```suggestion
    def __init__(self, name: str, default: Optional[Union[str, List[str]]] = None) -> None:
```